app/views/games/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<header class='masthead'>
+  <div class="container d-flex h-100 align-items-center">
+    <div class="mx-auto text-center">
+      <h1 class="mx-auto my-0 text-uppercase header-index">Game Library</h1></br>
+      <section class="top-margin">
+        <table class = "game-table">
+            <tr>
+              <th>Add to My Games</th>
+              <th></th>
+              <th>Title</th>
+              <th>Rating/100</th>
+              <th>Popularity</th>
+              <th>Themes</th>
+              <th>Video</th>
+            </tr>
+          <%@games.each do |game| %>
+              <tr id="game-<%= game.id %>">
+                <td><%= button_to user_games_path(game.id), method: :post, class: "transparent add-game" do %>
+                  <i class="fa fa-gamepad fa-4x", aria-hidden="true"></i>
```

Comment:
This is a new one for me, but you can call a `do` on an action helper, then pass something like an icon `<i>`